litmus-portal/frontend/src/views/MyHub/MyHubConnect/index.tsx

```diff
@@ -549,7 +549,7 @@ const MyHub = () => {
             {t('myhub.connectHubPage.videoDesc')}
           </Typography>
           <div className={classes.quickActionDiv}>
-            <QuickActionCard analyticsHome={false} nonAdmin />
+            <HomePageQuickActionCard />
```

Comment:
Can we use a single component named quick actions? from the generic components folder as it’s being reused at all places just with the change of data which may be switched using a prop depending on the page.

Reply:
Issue is not with the pros that is being passed to QuickActionCard but the way we want to pass the props. In Litmus, we can have a separate file with just data array specific to that page and then pass it to QuickActionCard. But for passing the text we require a functional component because we have to import usetransltion hook. For this every reason I took the quick action card from limtus-ui and then passed the props to it and made a local component specific to that page. If we keep the array in the same page file then the file will become big. Also for all pages having common QuickActionCard are using same local component. No redundant code.

Reply:
> In Litmus, we can have a separate file with just data array specific to that page and then pass it to QuickActionCard. But for passing the text we require a functional component because we have to import usetransltion hook. For this every reason I took the quick action card from limtus-ui and then passed the props to it and made a local component specific to that page. If we keep the array in the same page file then the file will become big. Also for all pages having common QuickActionCard are using same local component.

It’s not about redundant code, that way the code will be easier to manage as quick actions is not something feature related it’s generic thing which can hold context specific data, if the team approves then I am also fine with the current implementation too, though the suggested change would make it easy for community members to contribute.

Reply:
all requested changes implemented